src/Security/Security/help/Set-AzSecurityPricing.md

```diff
@@ -8,28 +8,35 @@ schema: 2.0.0
 # Set-AzSecurityPricing
 
 ## SYNOPSIS
-Sets the pricing of Azure Security Center tier for a scope.
+
+Sets the pricing data for the Azure Defender plans enabled in Azure Security Center for a scope.
```

Comment:
there are no scopes. should be same as get - 
Sets the Azure Defender plan for a subscription.